src/Shared/Const/Items.cs

```diff
@@ -52,28 +52,28 @@ public static int GetIndex(this InventoryCategory cat, int index)
 	/// </remarks>
 	public enum EquipSlot : byte
 	{
-		HairAccessory,
-		SubsidiaryAccessory,
-		Hair, // [i11025 (2016-02-26)]
-		_Outer1,
-		Top,
-		Costume,
-		Shoes,
-		_Helmet,
-		Armband,
-		LeftHand,
-		RightHand,
-		Gloves,
-		_Ring1,
-		_Ring2,
-		_Outer2,
-		Pants,
-		_Ring3,
-		_Ring4,
-		Bracelet1,
-		Bracelet2,
-		Necklace,
-		Lens,
+		HairAccessory, // HAT
+		SubsidiaryAccessory, // HAT_L
+		Hair, // HAIR
+		Top, // SHIRT
+		Gloves, // GLOVES
+		Shoes, // BOOTS
+		Helmet, // HELMET
+		Armband, // ARMBAND
+		RightHand, // RH
+		LeftHand, // LH
+		Outer1, // OUTER
+		Ring1, // OUTERADD1
+		Ring2, // OUTERADD2
+		Outer2, // BODY
+		Pants, // PANTS
+		Ring3, // PANTSADD1
+		Ring4, // PANTSADD2
+		Bracelet1, // RING1
+		Bracelet2, // RING2
+		Necklace, // NECK
+		Hat, // HAT_T
+		Lens, // LENS
```

Comment:
Hopefully the case on these comments don't bother you @exectails . I do want to keep them around because they change often enough, and having the correct names even as a reference will make it easy.